ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -8,8 +8,9 @@
 from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
     statistical_dtype_values,
 )
-from ivy_tests.test_ivy.test_functional.test_experimental.test_core.test_statistical\
-    import statistical_dtype_values as statistical_dtype_values_experimental
+from ivy_tests.test_ivy.test_functional.test_experimental.test_core.test_statistical import (

```

Comment:
Actually, just one small thing before I merge the PR, can you kindly fix the lint error that's being caused by this line. I think the original version was correct so you didnt have to change it

Reply:
Hi @YushaArif99, 
sure I'll change it to the old version.👍

Reply:
The auto-formatter changes the line, so I just committed it without running the formatter. I hope that's ok.